package.json

```diff
@@ -92,7 +92,6 @@
     "ember-cli-deploy": "^1.0.2",
     "ember-cli-deploy-build": "^2.0.0",
     "ember-cli-deploy-git": "^1.3.4",
-    "ember-cli-deploy-git-ci": "^1.0.1",
```

Comment:
I made the error of installing the addon without fully understanding the CI/CD workflow that had existed. Here's the list of packages "before":

https://github.com/ember-intl/ember-intl/blob/4a61ce6d7f64a29f15d5aced28015c16761c28fe/package.json#L80-L84

The CI/CD workflow had been such that we wouldn't need to use public and private SSH keys.